docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -216,45 +269,44 @@ Sebuah grup yang dihasilkan secara otomatis termasuk halaman dengan frontmatter
 		{
 			label: 'Panduan',
 			items: [
-				{ label: 'Sebuah halaman', link: '#' },
+				{ label: 'Sebuah halaman', link: '' },
 				{
 					label: 'Label sidebar custom',
-					link: '#',
+					link: '',
 					badge: { text: 'Baru', variant: 'tip' },
 				},
-				{ label: 'Halaman lain', link: '#' },
+				{ label: 'Halaman lain', link: '' },
 			],
 		},
 	]}
 />
 
-:::note
-Konfigurasi frontmatter `sidebar` hanya digunakan untuk tautan yang dihasilkan secara otomatis dan akan diabaikan untuk tautan yang ditentukan secara manual.
+:::catatan
+Konfigurasi frontmatter `sidebar` hanya digunakan untuk tautan dalam grup yang dibuat secara otomatis dan tautan dokumen yang didefinisikan dengan properti `slug`. Konfigurasi ini tidak berlaku untuk tautan yang didefinisikan dengan properti `link`.
 :::
 
 ## Badges
 
 Tautan, grup, dan grup yang dibuat secara otomatis juga dapat menyertakan properti `badge` untuk menampilkan _badge_ di samping labelnya.
 
-```js
+```js {9,16}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
+			label: 'Stars',
```

Comment:
This label can be safely translated and updated in the preview below.

Reply:
Ok, will update soon.